content/en/docs/setup/learning-environment/minikube.md

```diff
@@ -192,15 +192,16 @@ For example the command would be.
 ```shell
 minikube start --driver=<driver_name>
 ```
- Minikube supports the following drivers:
+Minikube supports the following drivers:
  {{< note >}}
  See [DRIVERS](https://minikube.sigs.k8s.io/docs/reference/drivers/) for details on supported drivers and how to install
 plugins.
 {{< /note >}}
 
-* virtualbox
+* docker ([driver installation](https://minikube.sigs.k8s.io/docs/drivers/docker/))
+* virtualbox ([driver installation](https://minikube.sigs.k8s.io/docs/drivers/virtualbox/))
+* Podman ([driver installation](https://minikube.sigs.k8s.io/docs/drivers/podman/)) (EXPERIMENTAL)
```

Comment:
Should this actually be:
```suggestion
* podman ([driver installation](https://minikube.sigs.k8s.io/docs/drivers/podman/)) (EXPERIMENTAL)
```
?

All the other driver names are lower case.

Reply:
Done incorporating suggestion.